pyperformance/run.py

```diff
@@ -136,6 +158,10 @@ def add_bench(dest_suite, obj):
 
             return dest_suite
 
+        if name in loops:
+            pyperf_opts.append(f"--loops={loops[name]}")
+            pyperf_opts.append("--warmups=1")
```

Comment:
Hard-coding this to `1` feels off.  IIRC, the number of warmups can impact results on PyPy.  Is it something we can also extract from the `--same-loops` JSON file?  If you're sure `1` is correct then I'm okay with that.

Reply:
Yeah, that's a good point.

Reply:
Looking at the data, it actually seems like we don't need this.

It looks like the first "run" of a benchmark has a number of warmups related to calibration -- as in it tries a bunch of times with an increasing loop number until it arrives at enough time:

```
          "warmups": [
            [1, 0.003556258001481183],
            [2, 0.0029844559976481833],
            [4, 0.0030109654981060885],
            [8, 0.003761511250559124],
            [16, 0.0030481524372589774],
            [32, 0.002865061812372005],
            [64, 0.0028616962342766783],
            [64, 0.002973547281271749],
            [64, 0.0028825146093822696],
            [64, 0.002945869875020435]
          ]
        },
```

These aren't really "warmups", they are calibration runs, but the data makes no distinction.

All subsequent runs just have one warmup:

```
          "warmups": [[64, 0.0028776703593393904]]
```

When you pass the `--loops` argument to `pyperf`, the whole "multiple warmups on the first run" thing goes away because it isn't doing calibration.

So, in short, I think I'll just take this line out.